code/modules/antagonists/abductor/equipment/gland.dm

```diff
@@ -93,10 +93,10 @@
 	hud.remove_atom_from_hud(gland_owner)
 	clear_mind_control()
 
-/obj/item/organ/heart/gland/mob_insert(mob/living/carbon/gland_owner, special = FALSE, movement_flags = DELETE_IF_REPLACED)
+/obj/item/organ/heart/gland/on_mob_insert(mob/living/carbon/gland_owner, special = FALSE, movement_flags)
 	. = ..()
 
-	if(special != 2 && uses) // Special 2 means abductor surgery
+	if(!special && uses) // Special 2 means abductor surgery
```

Comment:
You seem to have broken this code; the comment is true

![image](https://github.com/user-attachments/assets/a276cca1-353c-4f85-8e70-1c0709c6b7d1)

You should replace it with a define

Reply:
That should be an organ movement flag but sure.